src/test/scala/FizzBuzzSpecifications.scala

```diff
@@ -4,19 +4,19 @@ import org.scalacheck.{Gen, Properties}
 
 object FizzBuzzSpecifications extends Properties("FizzBuzz") {
   
-  val divBy3 = Gen.choose(1, 1000000) map(_ * 3) filter(_ % 5 != 0)
-  val divBy5 = Gen.choose(1, 1000000) map(_ * 5) filter(_ % 3 != 0)
-  val divBy15 = Gen.choose(1, 1000000) map(_ * 15)
+  val divOnlyBy3 = Gen.choose(1, 1000000) map(_ * 3) filter(_ % 5 != 0)
+  val divOnlyBy5 = Gen.choose(1, 1000000) map(_ * 5) filter(_ % 3 != 0)
+  val divBy3And5 = Gen.choose(1, 1000000) map(_ * 15)
```

Comment:
The `Only` part I don't like it. For instance, 30 is divisible by 3 but not `Only` by 3 but also by 10

Reply:
true.
Would you rather change the names to `divBy3NotBy5`?